rest/main_new.go

```diff
@@ -0,0 +1,53 @@
+package rest
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/couchbase/sync_gateway/base"
+	"github.com/imdario/mergo"
+	pkgerrors "github.com/pkg/errors"
+)
+
+// serverMainPersistentConfig runs the Sync Gateway server with persistent config.
+func serverMainPersistentConfig(osArgs []string, fs *flag.FlagSet, flagStartupConfig *StartupConfig) error {
+
+	// 3.0 config - bootstrap and pull configs from server buckets.
+	base.Infof(base.KeyAll, "Running in persistent config mode")
+
+	sc := DefaultStartupConfig(defaultLogFilePath)
+	base.Tracef(base.KeyAll, "default config: %#v", sc)
+
+	fileStartupConfig, err := LoadStartupConfigFromPaths(fs.Args()...)
```

Comment:
Is specifying multiple paths on startup a new feature/requirement?  If so, can you remind me of the details?  Doing a simple merge based on the order loaded seems like it would have corner cases that would result in error.

Reply:
Nope not a requirement, and agreed on the corner cases. Will remove.